src/Microsoft.AspNet.Mvc.Core/DefaultControllerActionArgumentBinder.cs

```diff
@@ -19,6 +19,9 @@ namespace Microsoft.AspNet.Mvc
     /// </summary>
     public class DefaultControllerActionArgumentBinder : IControllerActionArgumentBinder
     {
+        private static readonly MethodInfo CallPropertyAddRangeOpenGenericMethod =
+            typeof(MutableObjectModelBinder).GetTypeInfo().GetDeclaredMethod(nameof(CallPropertyAddRange));
```

Comment:
I duplicated the `private static` method from `MutableObjectModelBinder` to here.  different projects and it's only a few lines.